src/Components/Playground/Playground.tsx

```diff
@@ -20,15 +21,42 @@ const emptyPlayground = [
   [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
   [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
   [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
-  [0, 0, 0, 1, 0, 1, 0, 0, 0, 0],
-  [0, 0, 1, 0, 0, 0, 0, 0, 0, 0],
-  [1, 1, 1, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
+  [0, 0, 0, 0, 0, 0, 0, 0, 0, 0],
 ];
 export const Playground = () => {
-  const [playground] = useState(emptyPlayground);
-  const [figure] = useState(getRandomFigure());
+  const handleKeyPress = (e: KeyboardEvent) => {
+    const buttonKey = e.code;
+    console.log(e.code);
```

Comment:
Add the rule to not permit console.log in the main it will help in not adding debug code into main by mistake